provider/datagen/Cargo.toml

```diff
@@ -71,13 +71,13 @@ rayon = "1.5"
 rust-format = { version = "0.3.4", features = ["token_stream"] }
 serde = { version = "1.0", default-features = false, features = ["derive", "alloc"] }
 serde_json = { version = "1.0", default-features = false, features = ["alloc"] }
-serde-aux = { version = "2.1.1", default-features = false }
+serde-aux = { version = "4.1.2", default-features = false }
 syn = {version = "1.0", features = ["parsing"] }
 tinystr = { version = "0.7.1", path = "../../utils/tinystr", features = ["alloc", "serde", "zerovec"], default-features = false }
 toml = "0.5"
 writeable = { version = "0.5.1", path = "../../utils/writeable" }
 zerovec = { version = "0.9.2", path = "../../utils/zerovec", features = ["serde", "yoke"] }
-zip = { version = "0.5", default-features = false, features = ["deflate"] }
+zip = { version = "0.6", default-features = false, features = ["deflate"] }
```

Comment:
I'd rather not update `zip` to 0.6 just yet since we're trying to keep dependencies in sync with other places.